client/src/pages/admin/pages/__tests__/configuration.test.tsx

```diff
@@ -144,13 +143,7 @@ describe('Test Configuration', () => {
 
     nock('http://localhost:3000').put(`/api/config`).reply(200);
 
-    render(
-      <>
-        <Toaster />
```

Comment:
How come you're removing Toaster from these tests, does it work OK without?

Reply:
I did add toaster to the main wrapper that way the main wrapper will have it for every test to use